CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+<!-- MD024 - We want repeated headings in a changelog file -->
+<!-- markdownlint-disable-file MD024 -->
+
+# Changelog
+
+This changelog follows the patterns described here: <https://keepachangelog.com/en/1.0.0/>.
+
+Subheadings to categorize changes are `added, changed, deprecated, removed, fixed, security`.
```

Comment:
I quite like how wgpu handles their [instructions to contributors](https://github.com/gfx-rs/wgpu/blob/trunk/CHANGELOG.md?plain=1). That is, having them in a comment, so that they're visible if you're editing the file, but are otherwise hidden.

Reply:
fixed